deployment/v2/4_createRollup.ts

```diff
@@ -552,17 +552,22 @@ async function main() {
             emergencyBridgePauser: sovereignParams.emergencyBridgePauser,
         };
         genesis = await updateVanillaGenesis(genesis, chainID, initializeParams);
-        // Add weth address to deployment output if gas token address is provided and sovereignWETHAddress is not provided
+        // Add weth proxy and implementation address to deployment output if gas token address is provided and sovereignWETHAddress is not provided
         if (
             gasTokenAddress !== ethers.ZeroAddress &&
             ethers.isAddress(gasTokenAddress) &&
             (sovereignParams.sovereignWETHAddress === ethers.ZeroAddress ||
                 !ethers.isAddress(sovereignParams.sovereignWETHAddress))
         ) {
             const wethObject = genesis.genesis.find(function (obj) {
-                return obj.contractName == "WETH";
+                return obj.contractName == "WETH proxy";
```

Comment:
Those strings shpuld be in a separate file with just constants, so it can be reused on tooling/scripts later on.
So we do not need to copy/paste the string and miss some caps letter